config.go

```diff
@@ -33,6 +33,14 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
```

Comment:
How can one then place resource restrictions on an exec'ed process? Run a
new container?

On Wed, Mar 2, 2016 at 1:02 PM, Michael Crosby <notifications@github.com>
wrote:

> In config.go
> <https://github.com/opencontainers/specs/pull/329#discussion_r54790623>:
>
> > @@ -33,6 +33,14 @@ type Process struct {
> >  	// Cwd is the current working directory for the process and must be
> >  	// relative to the container's root.
> >  	Cwd string `json:"cwd"`
>
> cgroups is more of a container level feature instead of just one process
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/329/files#r54790623>.
>